packages/@navikt/prosessmeny/src/index.tsx

```diff
@@ -0,0 +1,37 @@
+import bem from '@navikt/bem-utils';
+import * as React from 'react';
+import './indexStyles';
+import Step, { StepType } from './step';
+
+interface StepProps {
+    label: string;
+    type?: StepType;
+    isActive?: boolean;
+    isFinished?: boolean;
+    isDisabled?: boolean;
+}
+
+interface ProsessmenyProps {
+    /**
+     * label: string;
+    type?: StepType ('warning' | 'success' | 'danger' | 'default');
```

Comment:
Regner med dette fint kan fjernes?

Reply:
Dette er kommentarer som kommer i storybook. Jeg la de der fordi om man bruker et interface for en prop så står bare navnet på interfacet som prop-type. Ved å inkludere dette får man en beskrivelse av hva interfacet inneholder